### DISCUSSION TITLE
Does Echelon need a specific version of 115 ESR?

### DISCUSSION THREAD

**Discussion by blatantist** [echelon-theme/echelon#118] [Q&A] [answered]
Firefox has many different versions of 115 ESR: 115.0 to 115.16.1
Firefox-native-controls lists specific versions of 115. Should I just download the most recent version that the patch works with?
Also, in the Readme, it should list [where exactly to download 115 ESR](https://ftp.mozilla.org/pub/firefox/releases/). I had to search for a bit to find the link because just searching "Firefox 115 ESR" brings you to a [page](https://www.mozilla.org/en-US/firefox/115.0esr/releasenotes/) that only has links to download the newest version of Firefox ESR.

**Comment by commandcontrolQ** [comment, marked as answer]
The latest version of 115 ESR, 115.16.1, has a patched xul.dll in the `firefox-native-controls` repository. You can download the latest ESR release of Firefox 115.

**Reply by commandcontrolQ** [reply]
As a rule of thumb, make sure that the version of Firefox you are downloading matches with the version of the patched xul.dll.

**Reply by blatantist** [reply]
Huh. Strange. When I installed Echelon earlier, the firefox-native-controls repo displayed the newest release as 115.9,1, but now it shows 115.16.1. I'll update it later as long as Firefox doesn't complain that my profile is out of date.
